### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#186] [Q&A] [answered]
https://github.com/codeman1o1/TIJK-Bot/blob/main/main.py#L43
Wat is dit? Discriminator? Klinkt heftig

**Comment by JustIanJ** [comment]
De link werkt volgens mij niet, het is L43 uit main.py

**Comment by karstenzip** [comment, marked as answer]
The link does work.
A discriminator is your tag in Discord. They named it that, not me.
E.g. wumpus#**0001**
**0001** is the discriminator

**Comment by JustIanJ** [comment]
ok thanks
